ResearchKit/Common/ORKStepHeaderView.m

```diff
@@ -114,10 +114,15 @@ - (instancetype)initWithFrame:(CGRect)frame {
             [self addSubview:_instructionLabel];
         }
         
+        [_captionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
+        [_instructionLabel setContentHuggingPriority:UILayoutPriorityDefaultLow forAxis:UILayoutConstraintAxisVertical];
```

Comment:
No, they are not strictly needed.  I was doing this to be consistent with the ``learnMoreButton`` getting its default hugging priority set here.